File: .tmp_c4000/batch_04_go/3492_maximum_containers_on_a_ship.go
// LeetCode 3492 - Maximum Containers on a Ship
// https://leetcode.com/problems/maximum-containers-on-a-ship/

func maxContainers(n int, w int, maxWeight int) int {
	cap := n * n
	byW := maxWeight / w
	if cap < byW {
		return cap
	}
	return byW
}

File: .tmp_py4013/batch_13_dump/3492_maximum_containers_on_a_ship.js
// LeetCode 3492 - Maximum Containers on a Ship
// https://leetcode.com/problems/maximum-containers-on-a-ship/

var maxContainers = function(n, w, maxWeight) {
    const cap = n * n;
    const byW = Math.floor(maxWeight / w);
    return cap < byW ? cap : byW;
};
